packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -70,6 +72,10 @@ export class ZipkinExporter implements SpanExporter {
     resultCallback: (result: ExportResult) => void
   ) {
     this._logger.debug('Zipkin exporter export');
+    if (this._shutdown) {
+      setImmediate(resultCallback, ExportResult.FAILED_NOT_RETRYABLE);
```

Comment:
I know that @obecny has talked about using the Zipkin exporter for the web at some point.

What would you think of instead using a browser-compatible (though I believe effectively the same) idiom: `setTimeout(() => resultCallback(ExportResult.FAILED_NOT_RETRYABLE))`?

Reply:
Done.
But if this should really support browsers then corresponding tests for the supported browsers shoud be added to verify it's working.

Reply:
Yes, for sure - a number of things would need to change. Just wanting to help reduce future effort of that if we end up doing it!